lib/components/steps/mapping/CollectionsTab.tsx

```diff
@@ -1,23 +1,32 @@
-import {useState} from 'react';
-import {Stack, Typography, Box, Button, Link} from '@mui/material';
+import { useState } from 'react';
+import { Stack, Typography, Box, Button, Link } from '@mui/material';
 import StyledCard from '../../common/StyledCard.tsx';
 import {useDataContext} from "../../../contexts/data/DataContext.ts";
+import {useUIContext} from '../../../contexts/ui/UIContext.ts';
 import ModalHeightWrapper from "../../common/ModalHeightWrapper.tsx";
+import Tour from '../../common/Tour.tsx';
+import { TourSteps, tutorial } from '../../common/tutorial.tsx';
 
 
 interface CollectionsProps {
     changeToNextTab: () => void;
     setDefaultCollection: (collectionId: string) => void
 }
 
-function CollectionsTab({changeToNextTab, setDefaultCollection}: CollectionsProps) {
+function CollectionsTab({ changeToNextTab, setDefaultCollection }: CollectionsProps) {
     const {
         collections,
     } = useDataContext();
+    const {
+        isTourOpen
+    } = useUIContext();
 
     const collectionKeys = Object.keys(collections);
 
     const [selectedCollection, setSelectedCollection] = useState<string>(collectionKeys.length > 0 ? collectionKeys[0] : '');
+    const [stepIndex, setStepIndex] = useState(0);
+
+    const onAfterChange = () => isTourOpen && setStepIndex(prevStepIndex => prevStepIndex + 1);
```

Comment:
note (non-blocking): As mentioned in a previous comment, the name of the function doesn't need to be equal to the name of the prop. In this case it would make it clear to the reader if onAfterChange were to be renamed updateTourStep f.e.